### PR TITLE
Replacing outdated information

### PR DESCRIPTION
**Pull request by AmmarYousefM**

### CODE REVIEW

File: app/help/en/index.md [hunk 1]
```diff
@@ -1,9 +1,15 @@
...
-[Cryptonomex](http://cryptonomex.com). You can see Graphene as a toolkit or
-technology for real-time blockchains.
+This is the reference wallet of BitShares Blockchain which connects to BitShares Blockchain nodes (bitshares-core).
+
+This reference wallet is a browser based wallet, which means your keys are stored locally in your browser and never leaves it while using the wallet. You should never expose your keys to anyone as transactions are signed locally before transmitting them to a node.
+
+BitShares Blockchain Reference wallet users can:
+
+- Login to BitShares Blockchain with their private keys to send and receive core cryptocurrency BTS.
+- Issue new tokens or utalize issued tokens at BitShares Blockchain.
```
Comment (abitmore) [inline app/help/en/index.md:10]: "utalize" typo?

Comment (AmmarYousefM) [reply app/help/en/index.md]: Fixed.

File: app/help/en/index.md [hunk 2]
```diff
@@ -1,9 +1,15 @@
...
-technology for real-time blockchains.
+This is the reference wallet of BitShares Blockchain which connects to BitShares Blockchain nodes (bitshares-core).
+
+This reference wallet is a browser based wallet, which means your keys are stored locally in your browser and never leaves it while using the wallet. You should never expose your keys to anyone as transactions are signed locally before transmitting them to a node.
+
+BitShares Blockchain Reference wallet users can:
+
+- Login to BitShares Blockchain with their private keys to send and receive core cryptocurrency BTS.
+- Issue new tokens or utilize issued tokens at BitShares Blockchain.
+- Send, receive, burn/issue and freeze/unfreeze issued tokens at BitShares Blockchain.
```
Comment (sschiessl-bcp) [inline app/help/en/index.md:11]: This list is not complete, should be formulated as such (or completely omitted)

File: app/help/en/index.md [hunk 3]
```diff
@@ -1,9 +1,15 @@
...
-[Graphene](https://github.com/cryptonomex/graphene) technology built by
-[Cryptonomex](http://cryptonomex.com). You can see Graphene as a toolkit or
-technology for real-time blockchains.
+This is the reference wallet of BitShares Blockchain which connects to BitShares Blockchain nodes (bitshares-core).
+
+This reference wallet is a browser based wallet, which means your keys are stored locally in your browser and never leaves it while using the wallet. You should never expose your keys to anyone as transactions are signed locally before transmitting them to a node.
+
+BitShares Blockchain Reference wallet users can:
+
+- Login to BitShares Blockchain with their private keys to send and receive core cryptocurrency BTS.
```
Comment (sschiessl-bcp) [inline app/help/en/index.md:9]: You can't really login to a blockchain, we should brainstorm what wording is appropriate

Comment (ioBanker) [reply app/help/en/index.md:9]: Perhaps replace with "login to BitShares Blockchain node (bitshares-core)"? please brainstorm or propose where/how to brainstorm.

File: app/help/en/index.md (hunk 2 above)
Comment (ioBanker) [reply app/help/en/index.md:11]: Please propose missing information.

**Pull request merged**